dex/msgjson/types.go

```diff
@@ -443,6 +443,7 @@ type Coin struct {
 type Prefix struct {
 	signable
 	AccountID     Bytes  `json:"accountid"`
+	Commitment    Bytes  `json:"com"`
```

Comment:
Despite the spec, I've put Commitment after ServerTime, and updated the spec in https://github.com/decred/dcrdex/pull/154